week-1/Homework/extra/1-currency-conversion.js

```diff
@@ -16,7 +20,12 @@ function convertToUSD() {}
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {// price as input
+  let onePercentOfPrice = (price*1)/100 // calculates transaction fee of 1%;
```

Comment:
Cool! You probably can leave out the `* 1` here. It won't do much when calculating 1% of  a number.

If you wanted 7% of a price, you would do `(price/100)* 7`. You find out 1% and multiply it by 7. Another way to do this is `price * 0.07` :)